NEWS.md

```diff
@@ -173,6 +173,8 @@
 
 29. `setkey()` now supports type `raw` as value columns (not as key columns), [#5100](https://github.com/Rdatatable/data.table/issues/5100). Thanks Hugh Parsonage for requesting, and Benjamin Schwendinger for the PR.
 
+30. `format_list_item.sfg()`, an unexported S3 method for the recently added `format_list_item()` generic, supports nice printing of simple feature geometry data.table columns, [#2273](https://github.com/Rdatatable/data.table/issues/2273).
```

Comment:
I would fold this into the earlier NEWS entry since both are improvements in dev